iFixit/react-commerce
# Community Page: Import from IFixit/ifixit

The transition to NextJS continues! Now that we have a route dedicated to `/Community`, let's add the (basic) page. This pull does not include support for the header and footer, since those will take some more work and will be done in another pull. 

Note: the only functional difference from `master` is the "Join the Community" button. On `master`, this displays a popup modal that the user can login/sign up through. We don't have an implementation of this in Next yet, so we are just redirecting to `/login` instead.

## CR
This is a pretty big pull. I did my best to break things up into small commits. The first commit is a straight copy/paste from IFixit/ifixit, then the following commits are my changes.

## QA
Make sure everything looks and behaves as `master` does. The only (current) exception is with the login/reset/register modals.

---

qa_req 2
